xml/System.Numerics/Complex.xml

```diff
@@ -1858,11 +1897,12 @@
         <Parameter Name="right" Type="System.Numerics.Complex" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">The double value to add.</param>
+        <param name="right">The complex value to add.</param>
+        <summary>Adds a double number to a complex number.</summary>
```

Comment:
```suggestion
        <summary>Adds a real number to a complex number.</summary>
```

Reply:
Based on @tannergooding 's suggestion above, lets change the summary to:

 ```suggestion
        <summary>Adds a double-precision real number to a complex number.</summary>
```